benchmarks/check.sh

```diff
@@ -47,7 +47,11 @@ run_coarse_prm ()
 }
 
 
-# run aspect on all .prm files in the current folder or any subdirectory
+# run aspect on all .prm files in the current folder or any
```

Comment:
```suggestion
# Run ASPECT on all .prm files in the current folder or any
```

Reply:
@tjhei: I can do that, but it would be the only comment in the whole file that satisfies this style. Which way do you want me to go?

Reply:
Well, you turned this into a sentence with a period at the end. I think it should be capitalized in that case. You can just push the "commit suggestion" button...